docs/canary-config.md

```diff
@@ -0,0 +1,99 @@
+Objects in this document are documented using [Markdown Syntax for Object Notation (MSON)].
+
+### Canary Config Object model (object)
+
+#### Properties
+- `name` **my-app golden signals canary config** (string, required) - Name for canary configuration.
```

Comment:
Could also add the `id` since it can now be specified on the config when it is pushed.

Reply:
Not sure how deep this should go, but: in orca, at least one of id and name (but not both) is required. If both are provided, name is ignored and id is used.

Reply:
- [ ] Add id

Reply:
@nisanharamati, I dug through the Kayenta code and it doesn't have any of that logic. It only basically toggles generating an id or not, which is what I have documented for now. However we can probably add a note in the description about Orca if you want.

Reply:
I think it's okay to leave out in here. I'll try and think of where would be a good place to document that behaviour.